tools/vipps-ecom-api-postman-collection.json

```diff
@@ -5,296 +5,6 @@
 		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
 	},
 	"item": [
-		{
-			"name": "Vipps Login",
```

Comment:
Here, I am removing the vipps login token from postman, because it's not needed any more. I moved the "user info" call down into the main section.